lib/utils.py

```diff
@@ -139,13 +148,30 @@ def decompress_buf(buf, out, des='file'):
                 out = f_in.read()
                 out = out.decode()
         else:
-            lStatus = False
             print("Unknow output type!")
-        return lStatus, out
+            return status
+        status = True
+        return status, out
     else:
-        lStatus = False
         print("Invalid input data!")
-        return lStatus
+        return status
+
+def message(meslevel, ty, content, level=1, log_file=None, **kwargs):
```

Comment:
why `**kwargs`?
Moreover, could you add in the docstring a bit more info about the various variables in the interface?